pkg/kubelet/kubelet_pods.go

```diff
@@ -675,6 +676,38 @@ func (kl *Kubelet) podIsTerminated(pod *v1.Pod) bool {
 	return false
 }
 
+func (kl *Kubelet) safeToDeletePod(pod *v1.Pod) bool {
+	if kubepod.IsMirrorPod(pod) {
```

Comment:
I dont have full context on how mirror pods work.  This is taken directly from the old status manager code.